README.md

```diff
@@ -39,14 +41,16 @@ this:
     b2 list-buckets
     b2 list-file-names <bucketName> [<startFileName>] [<maxToShow>]
     b2 list-file-versions <bucketName> [<startFileName>] [<startFileId>] [<maxToShow>]
+    b2 list-keys [--keyCount <maxKeyCount>] [--startKey <startApplicationKeyId>]
```

Comment:
Maybe this should be like `b2 ls` and just list them all, without the `--keyCount` and `--startKey` options.

Reply:
It's in the raw_api.py so I thought we would want to allow them to do this. 
I will take it out and you can decide some time.

`def list_keys(
        self,
        api_url,
        account_auth_token,
        account_id,
        max_key_count=None,
        start_application_key_id=None
    ):`

Reply:
I think the list-files and list-versions calls are useful only to people learning to call the APIS, they're not very useful for people who just want to list their files.  The same will be true for listing keys.  When people list their keys, they just want to list them all.